Files/style.css

```
@@ -0,0 +1,244 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+}
+
+body {
+  font-family: sans-serif;
+  font-size: 0.9rem;
+  font-weight: 3;
+  position: relative;
+  background: #f5f5f5;
+  background-color: #eceef1;
```

Also unnecessary, you are repeating yourself.
```suggestion
  background-color: #eceef1;
```